README-CN.md

```diff
@@ -90,6 +90,7 @@ const defaultPopperProps = {
 | Name                | Type         | DefaultValue                                 | Description  |
 | ------------------- | ------------ | -------------------------------------------- | ------------ |
 | `dayStr`            | `Array`      | `['日', '一', '二', '三', '四', '五', '六']`   | Used to set day name |
+| `monthStr`          | `Array`      | `['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12']`   | Used to set day name |
```

Comment:
Used to set month name

Reply:
Whoops! Copy/paste typos in several places! 🤦 I'll push an updated branch. 😃